webapp/src/reducers/index.js

```diff
@@ -193,6 +193,33 @@ const channelSubscriptions = (state = {}, action) => {
     }
 };
 
+const getIssueByKey = (state = {}, action) => {
+    switch (action.type) {
+    case ActionTypes.RECEIVED_JIRA_TICKETS :
+        return {
+            ...state,
+            isloaded: true,
+            assigneeName: action.data.fields.assignee && action.data.fields.assignee.displayName ? action.data.fields.assignee.displayName : '',
```

Comment:
might make sense to save `action.data.fields.assignee` in a var and reuse it here to improve readability.

Reply:
sure